package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "pbiviz": "pbiviz",
-    "start": "pbiviz start --target es6",
```

Comment:
please check package-lock.json. It shows 11,394 additions, but I don't see any changes here.